app/src/main/res/values/strings.xml

```diff
@@ -61,4 +61,5 @@
     <string name="favourite_icon">Favourite icon</string>
     <string name="month_day_dummy_text">Month Day</string>
     <string name="weekday_year_dummy_text">Weekday Year</string>
+    <string name="search_hint">Search apps...</string>
```

Comment:
Please change this to just `Search...`

Reply:
Are you sure about this? I don't think that will lead to good UX. 'Search...' does not indicate what the user is searching. Are contacts going to show up in the search? Are emails going to appear? Statistics for the apps? Contexts? 'Search Apps...' clearly hints at what the user can do with the widget.

Reply:
We are in an `All Apps` menu, you can only expect an application :smile: 
But do as you think it's better, it's not a crucial change and I'm ok with both approaches  :wink:

Reply:
I'm saying this because the google search bar searches for anything. It suggest contacts, apps and if you still submit the search it goes to the actual web search engine.